scintillometry/integration.py

```diff
@@ -406,6 +407,9 @@ def __init__(self, ih, n_phase, phase, average=True,
 
     def _read_frame(self, frame_index):
         # Read frame in phased directly, bypassing its ``read`` method.
+        print('Stack: time {}, sample {}/{}'
```

Comment:
Debugging statement?

Reply:
That is a debugging statement. I will remove it.